community/raspi2png/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: signageOS <dev@signageos.io>
+pkgname="raspi2png"
+pkgver="0.1"
+_commitid=5008e64ead4e1d3a6496ec98e800a695629a3a2e
+pkgrel=0
+pkgdesc="Utility to take a snapshot of the raspberry pi screen and save it as a PNG file"
+url="https://github.com/AndrewFromMelbourne/raspi2png"
+arch="armhf"
+license="MIT"
+depends=""
+makedepends="libpng-dev raspberrypi-dev"
+install=""
+options="!check"
+source="raspi2png-$pkgver.tar.gz::https://github.com/AndrewFromMelbourne/raspi2png/archive/$_commitid.tar.gz"
+builddir="$srcdir/"$pkgname-$_commitid
+
+build() {
+        cd "$builddir"
+        make
+}
+
+package() {
+        cd "$builddir"
```

Comment:
cd "$builddir" is not required unless your package is in main

Reply:
what do you mean? When the source .tar.gz is downloaded and extracted, there's a nested directory, the sources don't get extracted directly in the src root

Reply:
no need for cd "$builddir" statement because it is done automatically in the `prepare` , `build`, `check` and `package` functions.

Reply:
I see, ok thanks, fixed